Make popup table selection all input on creation

jessyec-s opened the pull request:
On popup table creation this patch sets the table's selected items to be all table input.  This default selection is kept until the user chooses another selection. This enables the chart to respond to actions in the table, such as sorting by column, when no selection has been specified by the user.

Commit:
Make popup table selection all input on creation

aptmac commented:
This approach works as long as there is a selection on the table, but not otherwise. What appears to be happening when the column header is clicked is that `buildChart()` (in `ChartAndPopupTableUI`) ultimately gets called and generates a new chart renderer that places the rows in the new order in the `threadRows` list in `ThreadsPage`. The selection changed listener doesn't appear to be hit when there isn't a chart selection, so the table will change the order of the displayed strings, but not the order of the underlying array.

![2019-10-07-chart-sorting](https://user-images.githubusercontent.com/10425301/66341693-d1812e80-e915-11e9-97c0-44a93c39f048.gif)

Commit:
Make chart build when column sort changes

jessyec-s commented:
I have changed my approach to this issue in 9b0c503.  Now I have added selection listeners to the columns in the popup table.  When a listener is triggered, if there are no selected rows in the table, the chart rebuilds.
